style.css

```diff
@@ -49,20 +64,37 @@
             background-position: center;
             background-size: cover;
             background-blend-mode: multiply;
-            height: 90vh;
-            position: relative;
+            height: 100vh;           
```

Comment:
Height dałem na 100vh, ponieważ to 90vh pewnie wydawało się ok ze względu na błąd, który powodowało wyśrodkowanie napisu w stuff(tworzył on na górze biały odstęp)